code.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html>

```

Comment:
Just a note, the Code page opens in a new tab when navigated to from the Home page, while all the other pages do not. Is this what we want?

Reply:
Thanks, fixed this so it opens in the same tab